index.js

```diff
@@ -96,6 +96,7 @@ let executeHooksWithArgs = (hooks = [], args) => {
                 commandIsRunning = _commandIsRunning
             }).run()
         } catch (e) {
+            reject(e)
```

Comment:
Then we only need to reject the error if it comes from Mocha. It should not affect the wdio hooks.